### DISCUSSION TITLE
Remove bindgen imports and exports from the wasm file.

### DISCUSSION THREAD

**Discussion by zie1ony** [wasm-bindgen/wasm-bindgen#3183] [Q&A]
The wasm file generated with `wasm_bindgen` contains all the `__wbindgen_placeholder__` stuff. Is it possible to remove all the bindgen imports and exports from the wasm file?

To see the motivation read: https://github.com/risc0/risc0/issues/299

**Comment by Liamolucko** [comment]
There isn't a proper way to do that.

If you want to do it manually, running the binary through the `wasm-bindgen` CLI first might actually be a good idea, since it gets rid of a bunch of things which are only there to pass info the the CLI. After that, there should only be imports left, which you can manually replace with stub functions that just contain `unreachable`. For example:

```diff
25c25,26
<   (import "./verify_bg.js" "__wbindgen_throw" (func $wasm_bindgen::__wbindgen_throw::h81a929e30b40f97f (type 5)))
---
>   (func $wasm_bindgen::__wbindgen_throw::h81a929e30b40f97f (type 5) (param i32 i32)
>     unreachable)
```
